src/include/daos_srv/pool.h

```diff
@@ -170,9 +179,27 @@ struct ds_pool_child {
 	void			*spc_metrics[DAOS_NR_MODULE];
 };
 
+struct ds_pool_svc_op_key {
+	uint64_t ok_client_time;
+	uuid_t   ok_client_id;
+	/* TODO: add a (cart) opcode to the key? */
+};
+
+struct ds_pool_svc_op_val {
+	int  ov_rc;
+	char ov_resvd[60];
+};
```

Comment:
What are these two extra changes for?

Reply:
These were one of the sections of the merge conflict.   I will remove them.